app/main/views/create_user.py

```diff
@@ -30,7 +30,10 @@ def create_user(encoded_token):
             "createuser.token_invalid: {encoded_token}",
             extra={'encoded_token': encoded_token}
         )
-        abort(400, INVALID_TOKEN_MESSAGE)
+        return render_template(
+            "toolkit/errors/400.html",
+            error_message=INVALID_TOKEN_MESSAGE,
+        ), 400
```

Comment:
Is there any reason we can't use render_error_page here?

Reply:
Kind of. Our `400` handler doesn't pass the error_message through to the template as it goes through `csrf_handler` first. Once we upgrade Flask past 0.11 we won't need to do that anymore.

Reply:
Okay, I guess we can put a TODO or record it as tech debt and come back to it later